src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,7 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+function grassman_pluecker_ideal(subspaces_dimension::Int, amb_dimension::Int)
+    return Polymake.ideal.pluecker_ideal(subspaces_dimension, amb_dimension)
```

Comment:
```suggestion
    return convert(Polymake.ideal.pluecker_ideal(subspaces_dimension, amb_dimension))::MPolyIdeal{fmpq_mpoly}
```
I think it would be good to return an Oscar ideal instead of a polymake type.

Reply:
This should also have a variant that takes a ring as input parameter and returns this as an ideal in this ring. Have a look at `toric_ideal` for example.